docs/tools/codetools/README.md

```diff
@@ -0,0 +1,14 @@
+# Use Flake8 for styling checks
+
+**Install Flake8**:
+https://flake8.pycqa.org/en/latest/
+
+```
+python -m pip install flake8
+```
+
+**Use Flake8 to check your code before submission**:
+
+```
+flake8 --config .flake8 [file_name]
```

Comment:
nit: anywhere in the labgraph directory, flake8 will find the closest config file (at the repo root) and use it. So you could also just have `flake8 [file_name]` or even `flake8 [directory_name]`